buildtools/country_data/filter_country_data.js

```diff
@@ -59,7 +59,7 @@ var translatableMsgs = fullCountryData.map(function(countryData) {
 console.log(translatableMsgs);
 
 // Make the JSON object reference the translatable messages.
-// This is kind of hacky since we set the the value to a string containing the
```

Comment:
duplicated words aren't technically spelling errors, but spell checkers have flagged them for decades...